### PR TITLE
Create BUILDING.md

### PR DESCRIPTION
**Pull request by adhavalboy**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+╔╗░╔╗░░╔╗╔╗░░░░╔╗╔╗╔╗░░░░╔╗░░╔╗
+║║░║║░░║║║║░░░░║║║║║║░░░░║║░░║║
+║╚═╝╠══╣║║║╔══╗║║║║║╠══╦═╣║╔═╝║
+║╔═╗║║═╣║║║║╔╗║║╚╝╚╝║╔╗║╔╣║║╔╗║
+║║░║║║═╣╚╣╚╣╚╝║╚╗╔╗╔╣╚╝║║║╚╣╚╝║
+╚╝░╚╩══╩═╩═╩══╝░╚╝╚╝╚══╩╝╚═╩══╝
+
+
+#compiler methods
```
Comment (Webd01) [inline BUILDING.md]: Can you write it like this, please?
```md
# Compiling
```

File: BUILDING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+║║░║║░░║║║║░░░░║║║║║║░░░░║║░░║║
+║╚═╝╠══╣║║║╔══╗║║║║║╠══╦═╣║╔═╝║
+║╔═╗║║═╣║║║║╔╗║║╚╝╚╝║╔╗║╔╣║║╔╗║
+║║░║║║═╣╚╣╚╣╚╝║╚╗╔╗╔╣╚╝║║║╚╣╚╝║
+╚╝░╚╩══╩═╩═╩══╝░╚╝╚╝╚══╩╝╚═╩══╝
+
+
+#compiler methods 
+
+C language:
```
Comment (Webd01) [inline BUILDING.md]: Also, use h2 for the name of the languages, please.
```md
## Language name
....
## Language name
...
```

File: BUILDING.md (hunk 1 above)
Comment (adhavalboy) [reply BUILDING.md]: done!

**Comment by Webd01** [comment]
Thanks!

**Pull request merged**